### PR TITLE
Update protobuf generated code. Expand README.

### PR DESCRIPTION
**Pull request by afs**
Align protobuf code with the recently dependabot updated runtime.

----

By submitting this pull request, I acknowledge that I am making a contribution to the Apache Software Foundation under the terms and conditions of the [Contributor's Agreement](https://www.apache.org/licenses/contributor-agreements.html).

### CODE REVIEW

File: jena-arq/Grammar/RDF-Protobuf/README [hunk 1]
```diff
@@ -1,6 +1,20 @@
 ## Licensed under the terms of http://www.apache.org/licenses/LICENSE-2.0
 
+An Apache Jena reklease includes the java generated by teh protobuf compiler.
```
Comment (kinow) [inline jena-arq/Grammar/RDF-Protobuf/README:3]: s/reklease/release

s/teh/the

File: jena-arq/Grammar/RDF-Protobuf/README [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+There is no need to regenerate this inorder to build Jena.
```
Comment (kinow) [inline jena-arq/Grammar/RDF-Protobuf/README:4]: s/inorder/in order

**Review by kinow — APPROVED** [review]
Some typos, but no blockers.

File: jena-arq/Grammar/RDF-Protobuf/README (hunk 1 above)
Comment (afs) [reply jena-arq/Grammar/RDF-Protobuf/README]: Ug - classic! - failed to write the editor buffer.

**Pull request merged**